content/docs/intro/vs/cloud_template_transpilers.md

```diff
@@ -8,65 +8,65 @@ menu:
-  a rich experience including diffs and previews of updates before they are made. Troposphere/CDK rely on
+  especially with organization-wide sharing of projects and stacks. This is closer to "GitHub for DevOps" and delivers
+  a rich experience including diffs and previews of updates before they are made. Troposphere/CDK relies on
   CloudFormation which is known to be more challenging in these areas.
 
-* Pulumi has a built-in configuration system that was designed to be super easy to use. Related, encrypted secrets
-  give you an easy way to integrate secrets management best practices for database passwords, tokens, etc. In
+* Pulumi has a built-in configuration system that is super easy to use. Related, encrypted secrets
+  give you an easy way to integrate secrets management best practices for database passwords, tokens, and other secrets. In
   contrast, AWS offers building block services like AWS KMS, AWS Secrets Manager, and AWS Systems Manager Parameter
-  Store, however using them in combination with one another in just the right way can be challenging. Pulumi leverages
+  Store. However, using them in combination with one another in just the right way can be challenging. Pulumi leverages
   underlying building block services in your target cloud, or even HashiCorp Vault, to deliver an easy experience
   with secrets management automatic best practices built-in.
 
 * Pulumi integrates with [a number of CI/CD providers]({{< relref "/docs/guides/continuous-delivery" >}}) and
-  source control systems out of the box, to deliver easy continuous delivery with systems you might already be using.
+  source control systems (SCMs) out of the box, for easy continuous delivery with systems you might already be using.
   Although CloudFormation can be used in this manner, it requires manual configuration, and is designed to work
   best with AWS's own CodeBuild/Pipeline products.
 
-* Pulumi integrates with your identity provider, including GitHub, GitLab, Atlassian, or
-  [any SAML/SSO 2.0 provider]({{< relref "/docs/guides/saml" >}}), such as Azure Active Directory, Google G Suite,
-  or Okta, for auditing and access controls using your existing enterprise systems of record. AWS CloudFormation can
+* Pulumi integrates with your identity provider---including GitHub, GitLab, Atlassian, or
+  [any SAML/SSO 2.0 provider]({{< relref "/docs/guides/saml" >}}) (such as Azure Active Directory, Google G Suite,
+  or Okta)---for auditing and access controls using your existing enterprise systems of record. AWS CloudFormation can
   be manually integrated with those systems with greater effort.
 
-* Pulumi can be [use custom state management]({{< relref "/docs/intro/concepts/state#self-managed-backend" >}}),
-  and offers a self-hosting option, for greater control, including "behind the firewall" on-premises and hybrid
+* Pulumi can use [custom state management]({{< relref "/docs/intro/concepts/state#self-managed-backend" >}})
```

Comment:
Thank you for catching this!

Reply:
It was @mikhailshilkov who caught it. ;)